README.md

```diff
@@ -602,7 +606,59 @@ mailersend.deleteRecipient({
   recipient_id: 'xxx'
 });
 ```
+## Templates
 
+### Get a list of templates
+
+```js
+const MailerSend = require("mailersend");
+
+const mailersend = new MailerSend({
+    api_key: "key",
+});
+
+mailersend.templateList({
+})
+  .then(response => response.json())
+  .then(data => {
+    console.log(data);
+  });
+```
+
+### Get template
+
+```js
+const MailerSend = require("mailersend");
+
+const mailersend = new MailerSend({
+    api_key: "key",
+});
+
+mailersend.template({
+  template_id: 'xxx'
+})
+  .then(response => response.json())
+  .then(data => {
+    console.log(data);
+  });
+```
+
+### Delete template
+
+```js
+const MailerSend = require("mailersend");
+
+const mailersend = new MailerSend({
+    api_key: "key",
+});
+
+mailersend.deleteTemplate({
+  template_id: 'xxx'
+})
+  .then(response => {
```

Comment:
You can keep it like other endpoints where they are not handling the response at all, like [here](https://github.com/mailersend/mailersend-nodejs/blob/main/examples/webhooks/delete.js#L10). But as you've mentioned it was not the case in the `Domain` endpoint. I think that was a mistake in the example. Later we need to go through all of the existing endpoints and make things consistent everywhere. But for the time being, you can do it like the `Webhook` endpoint.